furious/context/context.py

```diff
@@ -244,6 +249,10 @@ def add(self, target, args=None, kwargs=None, **options):
 
         target.update_options(_context_id=self.id)
 
+        if target.get_queue() == ASYNC_DEFAULT_QUEUE and (
+            self._default_queue != ASYNC_DEFAULT_QUEUE):
+            target.update_options(queue=self._default_queue)
```

Comment:
yeah, true true, I also think this is a little convoluted. It should just check if there is a self._default_queue. I'll have to add something to async to determine if a queue was actually set or not vs someone setting it to default.